etc/ProdOfflineConfiguration.py

```diff
@@ -640,6 +644,15 @@
                tape_node="T0_CH_CERN_MSS",
                disk_node="T2_CH_CERN",
                scenario=ppScenario)
+    
+RAWSKIM_DATASETS = ["ParkingDoubleMuonLowMass0-ReserveDMu"]
+for rawSkimDataset in RAWSKIM_DATASETS:
+    addDataset(tier0Config, rawSkimDataset,
+               do_reco=True,
+               write_dqm=True,
+               archival_node=None,
+               tape_node="T1_US_FNAL_MSS",
+               scenario=ppScenario)
 
 DATASETS = ["ParkingDoubleMuonLowMass1","ParkingDoubleMuonLowMass2",
```

Comment:
All the ParkingDoubleMuonLowMass PDs should have the RawSkim

Reply:
Looks good. The corresponding streams also need to be added to the SECOND_AGENT_STREAMS list: ParkingDoubleMuonLowMass[0-3]

Reply:
Also reco can be switched OFF for these reserved parking PDs

Reply:
Hi @jeyserma @LinaresToine 

To confirm, this should be for the real situation test, not permanent. Physics coordinators agree to discuss internally with PAGs first, to understand the situation, sorting the priority, and come back to O&C. This is already agreed in XEB last week and also this week.

Actually, I understand from the joint meeting that, we will test with replay. Do I misunderstand that?

Thx.

Reply:
Hi @srimanob 

The idea is to test it in a replay once the first collision runs arrive, and then put it directly in production once the replay is validated. Of course, this assumes that we can have this skim in production, which I thought was the case after various discussions. If you say we still need to wait for approval from management, we will still test it with a replay but not put it in production. After GL from management, it can be enabled for the next ERA 2024C at 1200 bunches.